types/react-native/index.d.ts

```diff
@@ -8465,8 +8465,8 @@ export namespace Animated {
     export function parallel(animations: Array<CompositeAnimation>, config?: ParallelConfig): CompositeAnimation;
 
     type Mapping = { [key: string]: Mapping } | AnimatedValue;
-    interface EventConfig {
-        listener?: ValueListenerCallback;
```

Comment:
Is this callback type unused now? If so, can you remove it?

Reply:
No, it is used by Animated.Value

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7f0cd433de22a10d890c2e02a6a8cf23f7397617/types/react-native/index.d.ts#L8248-L8253

Reply:
Cool, thanks! (Sorry for the laziness, was on the road with my phone)

Reply:
No problem, thanks!